diff --git a/ifopt_ipopt/cmake/FindIPOPT.cmake b/ifopt_ipopt/cmake/FindIPOPT.cmake
index 912f20d..f0d3595 100644
--- a/ifopt_ipopt/cmake/FindIPOPT.cmake
+++ b/ifopt_ipopt/cmake/FindIPOPT.cmake
@@ -40,7 +40,7 @@
 # (To distribute this file outside of YCM, substitute the full
 #  License text for the above reference.)
 
-if(NOT WIN32)
+# if(NOT WIN32)
 
   # First priority is finding the package using IPOPT_DIR if set
   if(DEFINED ENV{IPOPT_DIR})
@@ -125,104 +125,104 @@ if(NOT WIN32)
   set(IPOPT_LINK_FLAGS "")
 
 # Windows platforms
-else()
-  include(SelectLibraryConfigurations)
-
-  set(IPOPT_DIR $ENV{IPOPT_DIR} CACHE PATH "Path to IPOPT build directory")
-
-  set(IPOPT_INCLUDE_DIRS ${IPOPT_DIR}/include/coin)
-  find_library(IPOPT_IPOPT_LIBRARY_RELEASE libipopt ${IPOPT_DIR}/lib
-                                                    ${IPOPT_DIR}/lib/coin
-                                                    NO_DEFAULT_PATH)
-  find_library(IPOPT_IPOPT_LIBRARY_DEBUG   libipoptD ${IPOPT_DIR}/lib
-                                                     ${IPOPT_DIR}/lib/coin
-                                                     NO_DEFAULT_PATH)
-
-  select_library_configurations(IPOPT_IPOPT)
-  set(IPOPT_LIBRARIES ${IPOPT_IPOPT_LIBRARY})
-
-  # Some old version of binary releases of IPOPT have Intel fortran
-  # libraries embedded in the library, newer releases require them to
-  # be explicitly linked.
-  if(IPOPT_IPOPT_LIBRARY)
-    get_filename_component(_MSVC_DIR "${CMAKE_LINKER}" DIRECTORY)
-
-    # Find the lib.exe executable
-    find_program(LIB_EXECUTABLE
-                 NAMES lib.exe
-                 HINTS "${_MSVC_BINDIR}"
-                       "C:/Program Files/Microsoft Visual Studio 10.0/VC/bin"
-                       "C:/Program Files (x86)/Microsoft Visual Studio 10.0/VC/bin"
-                       "C:/Program Files/Microsoft Visual Studio 11.0/VC/bin"
-                       "C:/Program Files (x86)/Microsoft Visual Studio 11.0/VC/bin"
-                       "C:/Program Files/Microsoft Visual Studio 12.0/VC/bin"
-                       "C:/Program Files (x86)/Microsoft Visual Studio 12.0/VC/bin"
-                       "C:/Program Files/Microsoft Visual Studio 14.0/VC/bin"
-                       "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin"
-                 DOC "Path to the lib.exe executable")
-    mark_as_advanced(LIB_EXECUTABLE)
-
-    # backup PATH environment variable
-    set(_path $ENV{PATH})
-
-    # Add th MSVC "Common7/IDE" dir containing the dlls in the PATH when needed.
-    get_filename_component(_MSVC_LIBDIR "${_MSVC_BINDIR}/../../Common7/IDE" ABSOLUTE)
-    if(NOT EXISTS "${_MSVC_LIBDIR}")
-      get_filename_component(_MSVC_LIBDIR "${_MSVC_BINDIR}/../../../Common7/IDE" ABSOLUTE)
-    endif()
-
-    if(EXISTS "${_MSVC_LIBDIR}")
-      set(_MSVC_LIBDIR_FOUND 0)
-      file(TO_CMAKE_PATH "$ENV{PATH}" _env_path)
-      foreach(_dir ${_env_path})
-        if("${_dir}" STREQUAL ${_MSVC_LIBDIR})
-          set(_MSVC_LIBDIR_FOUND 1)
-        endif()
-      endforeach()
-      if(NOT _MSVC_LIBDIR_FOUND)
-        file(TO_NATIVE_PATH "${_MSVC_LIBDIR}" _MSVC_LIBDIR)
-        set(ENV{PATH} "$ENV{PATH};${_MSVC_LIBDIR}")
-      endif()
-    endif()
-
-    if(IPOPT_IPOPT_LIBRARY_RELEASE)
-      set(_IPOPT_LIB ${IPOPT_IPOPT_LIBRARY_RELEASE})
-    else()
-      set(_IPOPT_LIB ${IPOPT_IPOPT_LIBRARY_DEBUG})
-    endif()
-
-    execute_process(COMMAND ${LIB_EXECUTABLE} /list "${_IPOPT_LIB}"
-                    OUTPUT_VARIABLE _lib_output)
-
-    set(ENV{PATH} "${_path}")
-    unset(_path)
-
-    if(NOT "${_lib_output}" MATCHES "libifcoremd.dll")
-      get_filename_component(_IPOPT_IPOPT_LIBRARY_DIR "${_IPOPT_LIB}" DIRECTORY)
-
-      foreach(_lib ifconsol
-                   libifcoremd
-                   libifportmd
-                   libmmd
-                   libirc
-                   svml_dispmd)
-        string(TOUPPER "${_lib}" _LIB)
-        find_library(IPOPT_${_LIB}_LIBRARY_RELEASE ${_lib} ${_IPOPT_IPOPT_LIBRARY_DIR})
-        find_library(IPOPT_${_LIB}_LIBRARY_DEBUG ${_lib}d ${_IPOPT_IPOPT_LIBRARY_DIR})
-        select_library_configurations(IPOPT_${_LIB})
-        list(APPEND IPOPT_LIBRARIES ${IPOPT_${_LIB}_LIBRARY})
-      endforeach()
-    endif()
-  endif()
-
-  set(IPOPT_DEFINITIONS "")
-  if(MSVC)
-    set(IPOPT_LINK_FLAGS "/NODEFAULTLIB:libcmt.lib;libcmtd.lib")
-  else()
-    set(IPOPT_LINK_FLAGS "")
-  endif()
-
-endif()
+# else()
+#   include(SelectLibraryConfigurations)
+
+#   set(IPOPT_DIR $ENV{IPOPT_DIR} CACHE PATH "Path to IPOPT build directory")
+
+#   set(IPOPT_INCLUDE_DIRS ${IPOPT_DIR}/include/coin)
+#   find_library(IPOPT_IPOPT_LIBRARY_RELEASE libipopt ${IPOPT_DIR}/lib
+#                                                     ${IPOPT_DIR}/lib/coin
+#                                                     NO_DEFAULT_PATH)
+#   find_library(IPOPT_IPOPT_LIBRARY_DEBUG   libipoptD ${IPOPT_DIR}/lib
+#                                                      ${IPOPT_DIR}/lib/coin
+#                                                      NO_DEFAULT_PATH)
+
+#   select_library_configurations(IPOPT_IPOPT)
+#   set(IPOPT_LIBRARIES ${IPOPT_IPOPT_LIBRARY})
+
+#   # Some old version of binary releases of IPOPT have Intel fortran
+#   # libraries embedded in the library, newer releases require them to
+#   # be explicitly linked.
+#   if(IPOPT_IPOPT_LIBRARY)
+#     get_filename_component(_MSVC_DIR "${CMAKE_LINKER}" DIRECTORY)
+
+#     # Find the lib.exe executable
+#     find_program(LIB_EXECUTABLE
+#                  NAMES lib.exe
+#                  HINTS "${_MSVC_BINDIR}"
+#                        "C:/Program Files/Microsoft Visual Studio 10.0/VC/bin"
+#                        "C:/Program Files (x86)/Microsoft Visual Studio 10.0/VC/bin"
+#                        "C:/Program Files/Microsoft Visual Studio 11.0/VC/bin"
+#                        "C:/Program Files (x86)/Microsoft Visual Studio 11.0/VC/bin"
+#                        "C:/Program Files/Microsoft Visual Studio 12.0/VC/bin"
+#                        "C:/Program Files (x86)/Microsoft Visual Studio 12.0/VC/bin"
+#                        "C:/Program Files/Microsoft Visual Studio 14.0/VC/bin"
+#                        "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin"
+#                  DOC "Path to the lib.exe executable")
+#     mark_as_advanced(LIB_EXECUTABLE)
+
+#     # backup PATH environment variable
+#     set(_path $ENV{PATH})
+
+#     # Add th MSVC "Common7/IDE" dir containing the dlls in the PATH when needed.
+#     get_filename_component(_MSVC_LIBDIR "${_MSVC_BINDIR}/../../Common7/IDE" ABSOLUTE)
+#     if(NOT EXISTS "${_MSVC_LIBDIR}")
+#       get_filename_component(_MSVC_LIBDIR "${_MSVC_BINDIR}/../../../Common7/IDE" ABSOLUTE)
+#     endif()
+
+#     if(EXISTS "${_MSVC_LIBDIR}")
+#       set(_MSVC_LIBDIR_FOUND 0)
+#       file(TO_CMAKE_PATH "$ENV{PATH}" _env_path)
+#       foreach(_dir ${_env_path})
+#         if("${_dir}" STREQUAL ${_MSVC_LIBDIR})
+#           set(_MSVC_LIBDIR_FOUND 1)
+#         endif()
+#       endforeach()
+#       if(NOT _MSVC_LIBDIR_FOUND)
+#         file(TO_NATIVE_PATH "${_MSVC_LIBDIR}" _MSVC_LIBDIR)
+#         set(ENV{PATH} "$ENV{PATH};${_MSVC_LIBDIR}")
+#       endif()
+#     endif()
+
+#     if(IPOPT_IPOPT_LIBRARY_RELEASE)
+#       set(_IPOPT_LIB ${IPOPT_IPOPT_LIBRARY_RELEASE})
+#     else()
+#       set(_IPOPT_LIB ${IPOPT_IPOPT_LIBRARY_DEBUG})
+#     endif()
+
+#     execute_process(COMMAND ${LIB_EXECUTABLE} /list "${_IPOPT_LIB}"
+#                     OUTPUT_VARIABLE _lib_output)
+
+#     set(ENV{PATH} "${_path}")
+#     unset(_path)
+
+#     if(NOT "${_lib_output}" MATCHES "libifcoremd.dll")
+#       get_filename_component(_IPOPT_IPOPT_LIBRARY_DIR "${_IPOPT_LIB}" DIRECTORY)
+
+#       foreach(_lib ifconsol
+#                    libifcoremd
+#                    libifportmd
+#                    libmmd
+#                    libirc
+#                    svml_dispmd)
+#         string(TOUPPER "${_lib}" _LIB)
+#         find_library(IPOPT_${_LIB}_LIBRARY_RELEASE ${_lib} ${_IPOPT_IPOPT_LIBRARY_DIR})
+#         find_library(IPOPT_${_LIB}_LIBRARY_DEBUG ${_lib}d ${_IPOPT_IPOPT_LIBRARY_DIR})
+#         select_library_configurations(IPOPT_${_LIB})
+#         list(APPEND IPOPT_LIBRARIES ${IPOPT_${_LIB}_LIBRARY})
+#       endforeach()
+#     endif()
+#   endif()
+
+#   set(IPOPT_DEFINITIONS "")
+#   if(MSVC)
+#     set(IPOPT_LINK_FLAGS "/NODEFAULTLIB:libcmt.lib;libcmtd.lib")
+#   else()
+#     set(IPOPT_LINK_FLAGS "")
+#   endif()
+
+# endif()
 
 mark_as_advanced(IPOPT_INCLUDE_DIRS
                  IPOPT_LIBRARIES
